lib/gui/modules/image-writer.js

```diff
@@ -63,6 +66,15 @@ imageWriter.service('ImageWriterService', function ($q, $rootScope) {
       })
       child.on('error', reject)
       child.on('done', resolve)
+      child.on('timeout', (timeoutData) => {
+        analytics.logEvent('Writer timeout', timeoutData)
+        // eslint-disable-next-line no-magic-numbers
+        const timeoutDataSeconds = _.update(timeoutData, [ 'timeout' ], _.partial(_.divide, _, 1000))
```

Comment:
Should we create a utility function (maybe in `lib/shared/utils`) to transform milliseconds to seconds?

Reply:
Done!